src/odemis/acq/test/stream_test.py

```diff
@@ -3099,8 +3015,30 @@ def test_acq_arpol(self):
             # estimated acquisition time should be accurate with less than 50% margin + 1 extra second
             timeout = 1 + 1.5 * sas.estimateAcquisitionTime()
             start = time.time()
+            im0 = ars.image.value
             f = sas.acquire()
 
+            if pos == "acquireAllPol":
+                # Check if there is a live update in the setting stream.
+                # Only checked with multiple polarization, as otherwise, there is
+                # only one image anyway.
```

Comment:
This is only true if one polarization position and no integration time correct? If there is 1 pol pos and integration count >1 then there should be a lso a live update with different content?

Reply:
Yes, but as it's a 1x1 acquisition, it's not really possible to check that the live image "updates", as there is only one. So we only check where it's easy: when the polarization changes.